src/Lightest.Data.Mongo/Models/Services/IUploadDataRepository.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+namespace Lightest.Data.Mongo.Models.Services
+{
```

Comment:
Resolve merge conflict in src/Lightest.Data.Mongo/Models/Services/IUploadDataRepository.cs